leaf-ui/js/object/Class.js

```diff
@@ -338,12 +346,26 @@ class AnalysisResult {
         this.timePointPathSize;
         this.elementList; 
         this.allSolution;
```

Comment:
I'm guessing this is the allSolution param you mentioned in your message -- I'll do some testing but for now we can just leave it there, could you maybe add a comment on it that just says like `// potentially deprecated` or something like that?